### PR TITLE
Python quickcheck

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

**Comment by Keats** [comment]
would it be possible to have the example call an imported function from a module? is it possible?

**Comment by teh** [comment]
Done. Also added an embedded tweet to see what it looks like (good in our stylesheet so thats nice)

File: content/articles/haskell-quickcheck-in-python.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+```python
+reverse(reverse(x)) == x
+```
+
+QuickCheck will try to find a counterexample to that proposition by
+generating random values for `x`. And it is very good at finding mean values:
+
+<blockquote class="twitter-tweet" lang="en"><p>Haskell Quickcheck enters a bar: asks for 1 beer, 42 beers, -Inifinity beers, shaves bartenders beard, sets off a tactical nuke.</p>&mdash; Michael Neale (@michaelneale) <a href="https://twitter.com/michaelneale/status/567532684595851264">February 17, 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (Keats) [inline content/articles/haskell-quickcheck-in-python.md]: do we really want to have the page depend on js?

Comment (teh) [inline content/articles/haskell-quickcheck-in-python.md]: This degrades gracefully like our commenting system. I.e. it will still render without js

**Pull request merged**